pycontrails/models/ps_model/ps_model.py

```diff
@@ -0,0 +1,936 @@
+        transport aircraft in the cruise. Part 1: fundamental quantities and governing relations for a general
+        atmosphere. Aero. J., 125(1284), 296-340, doi: 10.1017/aer.2020.62.
+    """
+    mu = fluid_dynamic_viscosity(air_temperature)
+    return (
+        wing_surface_area**0.5
+        * mach_num
+        * (air_pressure / mu)
+        * (constants.kappa / (constants.R_d * air_temperature))**0.5
+    )
+
+
+def fluid_dynamic_viscosity(air_temperature: npt.NDArray[np.float_]) -> npt.NDArray[np.float_]:
+    """
+    Calculate fluid dynamic viscosity.
+
+    Parameters
+    ----------
+    air_temperature : npt.NDArray[np.float_]
+        Ambient temperature at each waypoint, [:math:`K`]
+
+    Returns
+    -------
+    npt.NDArray[np.float_]
+        Fluid dynamic viscosity, [:math:`kg m^{-1} s^{-1}`]
+
+    Notes
+    -----
+    The dynamic viscosity is a measure of the fluid's resistance to flow and is represented by Sutherland's Law.
+    The higher the viscosity, the thicker the fluid.
+
+    References
+    ----------
+    - Refer to Eq. (25) of Poll & Schumann (2021).
+    - Poll & Schumann (2021). An estimation method for the fuel burn and other performance characteristics of civil
+        transport aircraft in the cruise. Part 1: fundamental quantities and governing relations for a general
+        atmosphere. Aero. J., 125(1284), 296-340, doi: 10.1017/aer.2020.62.
+    """
+    return 1.458E-6 * (air_temperature**1.5) / (110.4 + air_temperature)
+
+
+# -------------------------------
+# Lift and drag coefficients
+# -------------------------------
+
+
+def lift_coefficient(
+        wing_surface_area: float,
+        aircraft_mass: npt.NDArray[np.float_],
+        air_pressure: npt.NDArray[np.float_],
+        mach_num: npt.NDArray[np.float_],
+        climb_angle: npt.NDArray[np.float_] | None = None,
+) -> npt.NDArray[np.float_]:
+    """Calculate the lift coefficient.
+
+    This quantity is a dimensionless coefficient that relates the lift generated
+    by a lifting body to the fluid density around the body, the fluid velocity,
+    and an associated reference area.
+
+    Parameters
+    ----------
+    wing_surface_area : float
+        Aircraft wing surface area, [:math:`m^2`]
+    aircraft_mass : npt.NDArray[np.float_]
+        Aircraft mass, [:math:`kg`]
+    air_pressure: npt.NDArray[np.float_]
+        Ambient pressure, [:math:`Pa`]
+    mach_num : npt.NDArray[np.float_]
+        Mach number at each waypoint
+    climb_angle : npt.NDArray[np.float_] | None
+        Angle between the horizontal plane and the actual flight path, [:math:`\deg`]
+
+    Returns
+    -------
+    npt.NDArray[np.float_]
+        Lift coefficient
+
+    Notes
+    -----
+    The lift force is perpendicular to the flight direction, while the aircraft weight acts vertically.
+    """
+    if climb_angle is None:
+        climb_angle = np.zeros_like(aircraft_mass)
```

Comment:
Perhaps this isn't needed -- You may want `climb_angle` take a default value of 0.0 (instead of None), then avoid creating a constant 1D array here.